src/storage/index/LookupBaseProcessor.h

```diff
@@ -81,6 +81,7 @@ class LookupBaseProcessor : public BaseProcessor<RESP> {
   // Save schemas when column is out of index, need to read from data
   std::vector<std::shared_ptr<const meta::NebulaSchemaProvider>> schemas_;
   std::vector<size_t> deDupColPos_;
+  int64_t limit_ = -1;
```

Comment:
maybe the default value of `limit `is 0 (0 means no limit) better?
This is not important